packages/dx-react-grid/src/plugins/grouping-state.test.jsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
```

Comment:
I don't like an idea to mix shallow and mount for the plugins. We should introduce a new way to test plugins with shallow. It is not as trivial as you used here. Let's use the only mount in plugins tests. Or switch to shallow completely.

Reply:
😣 let's keep mount for now. 'switch to shallow completely' - it's too big task for this PR.